src/main/java/com/automatedtest/sample/driver/DriverUtils.java

```diff
@@ -5,34 +5,70 @@
 import org.openqa.selenium.OutputType;
 import org.openqa.selenium.TakesScreenshot;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.firefox.FirefoxProfile;
 
 import java.io.File;
 
 public class DriverUtils {
-    private WebDriver driver;
+    public static ThreadLocal<WebDriver> currentDriver = new ThreadLocal<WebDriver>();
     private Wait wait;
 
-    public DriverUtils(WebDriver driver) {
-        this.driver = driver;
-        this.wait = new Wait(driver);
+    public DriverUtils() {
+        this.wait = new Wait(this.currentDriver.get());
+    }
+
+    public static void driverInit() {
+        WebDriver driver;
+
+        String browser = System.getProperty("browser");
+        if (browser == null) {
+            browser = "chrome";
+        }
+        switch (browser) {
+            case "chrome":
+                ChromeOptions chromeOptions = new ChromeOptions();
+                chromeOptions.addArguments("['start-maximized']");
+                chromeOptions.addArguments("--lang=en-GB");
+                driver = new ChromeDriver(chromeOptions);
+                break;
+            case "firefox":
+                FirefoxProfile profile = new FirefoxProfile();
+                profile.setPreference("intl.accept_languages","en-GB");
+                FirefoxOptions options = new FirefoxOptions();
+                options.setProfile(profile);
+                driver = new FirefoxDriver(options);
+                break;
+            default:
+                throw new IllegalArgumentException("Browser \"" + browser + "\" isn't supported.");
+        }
+        driver.manage().window().maximize();
+        currentDriver.set(driver);
     }
 
     public void scrollToBottom() {
```

Comment:
since right now we use driver as a static member, we should convert this and other method in this class static as well